docs/metrics.md

```diff
@@ -60,5 +63,83 @@ on the Google cloud project.
     5. Example graph for fs/ops_count
 ![fs/ops_count](https://user-images.githubusercontent.com/101323867/188802087-6423f4f1-2aa6-4501-8db6-3d1997986f68.png)
 
+## Prometheus metrics
+
+1. Enable Prometheus metrics exporter using the configuration file. The Prometheus metrics endpoint will be exposed on this port and a path of `/metrics`. For example:
+
+```yaml
+monitoring:
+  prometheus-port: 8080
+```
+
+2. Lanuch GCSFuse, for example:
+
+```bash
+gcsfuse --config-file <config_file_path> <bucket_name> <directory_name>
+```
+
+3. Run the following command to validate the Prometheus metrics endpoint is available.
+
+```bash
+curl http://localhost:8080/metrics
+```
+
+The output is similar to the following:
+
+```text
+# HELP file_cache_read_bytes_count The cumulative number of bytes read from file cache along with read type - Sequential/Random
+# TYPE file_cache_read_bytes_count counter
+file_cache_read_bytes_count{read_type="Random"} 0
```

Comment:
This might be coming from somewhere else, but could we prefix all metrics names with "gcsfuse_", for example 
gcsfuse_file_cache_read_bytes_count ?

Reply:
I'd like to keep this PR focusing on the prometheus integration. I can create a follow-up PR to modify the metrics name, but I recommend GCSFuse team make such changes because it will introduce compatibility issues as the current test pipelines may rely on these metrics names.